SubmitHGCalPGun.py

```diff
@@ -155,6 +168,31 @@ def submitHGCalProduction():
     DASquery=False
     if opt.RELVAL != '':
         DASquery=True
+        
+    # in case of PU, GSD needs the MinBias
+    if int(opt.PU) != 0:
+        # we need to get the PU dataset
+        puname=str(opt.PUDS).split('/')[1]
+        # PLEASE NOTE --> using only 20 MinBias files here!! change to to 10000 if you want them all
+        cmd='das_client --limit 20 --query="file dataset='+str(opt.PUDS)+'" | grep '+puname
```

Comment:
Hi Luca,
Does it make sense to make this configurable as well or for how many generated events do you think that 20 files are sufficient?
Thank,
Clemens